examples/controlnet/train_controlnet.py

```diff
@@ -818,8 +847,19 @@ def load_model_hook(models, input_dir):
         accelerator.register_save_state_pre_hook(save_model_hook)
```

Comment:
it seems like current implementations of load_model_hook/save_model_hook won't handle saving of unet weights